Bump @audius/sdk version to 1.0.26

dharit-tan opened the pull request:
### Description
Bump `@audius/sdk` version

### Dragons

### How Has This Been Tested?

### How will this change be monitored?

### Feature Flags ###

audius-infra commented:
Preview this change https://demo.audius.co/rt-sdk

sliptype reviewed (approved):
There is an npm script in the root that will automatically do this for you btw!

dharit-tan commented:
> There is an npm script in the root that will automatically do this for you btw!

Yeah I saw! I tried it but it also updated a different package and changed the shas, so I decided to just bump manually.
